test/integration/manifests/swiftv2/mtpod1.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    kubernetes.azure.com/pod-network: aksswiftvnetv20425
```

Comment:
rename as pn1?

Reply:
It's not that easy to rename since it's the dummy cluster and all the things have been set up already. If I rename it, I think I have to recreate the whole dummy cluster stuff.